projects/sfl/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@cleo-cic/sfl",
```

Comment:
This needs to be changed to match the `npm` package name, right?

Reply:
Similar to the other comment, we don't have a npm scope yet, so I'll name this ngx-json-schema-form for now.